.github/libs/versionUpdater.js

```diff
@@ -0,0 +1,98 @@
+const semanticVersionLevels = {
+    major: 'MAJOR',
+    minor: 'MINOR',
```

Comment:
Please rename this constant to `UPPER_SNAKE_CASE`

```suggestion
    MINOR: 'MINOR',
```